src/realm.js

```diff
@@ -476,10 +477,10 @@ export class Realm {
     }
     context.savedEffects = [
       this.intrinsics.undefined,
-      this.generator,
-      this.modifiedBindings,
-      this.modifiedProperties,
-      this.createdObjects,
+      (this.generator: any),
```

Comment:
Why the new sad : any casts?

Reply:
Flow now catches this outrage. Yay. There is an issue for fixing it, but not a priority right now.